spark-bigquery-dsv2/spark-bigquery-spark24/src/test/java/com/google/cloud/spark/bigquery/acceptance/Spark24DataprocImage15AcceptanceTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spark.bigquery.acceptance;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+public class Spark24DataprocImage15AcceptanceTest extends DataprocAcceptanceTestBase {
+
+  private static AcceptanceTestContext context;
+
+  public Spark24DataprocImage15AcceptanceTest() {
+    super(context, false);
+  }
+
+  @BeforeClass
+  public static void setup() throws Exception {
+    context = DataprocAcceptanceTestBase.setup("1.5-debian10");
```

Comment:
If we are keeping the enum DataprocImage, then we can get this String from the enum